Formula/podman.rb

```diff
@@ -29,6 +48,20 @@ def install
       "linux"
     end
 
+    # Install podman, only on Linux
+    on_linux do
+      resource("cni-plugins").stage do
```

Comment:
these were the minimum plugins I needed for `podman` to be happy, should these be moved to a separate formula?

Reply:
Most likely